spec.html

```diff
@@ -245,62 +252,62 @@ <h3>Non-terminals</h3>
                   <tr class="section">
                     <td class="fix nt">subtype</td>
                     <td class="fix op">::=</td>
-                    <td class="fix ex">"\x00"</td>
+                    <td class="fix ex">null_byte</td>
                     <td >Generic binary subtype</td>
                   </tr>
                   <tr>
                     <td class="fix nt"></td>
                     <td class="fix op">|</td>
-                    <td class="fix ex">"\x01"</td>
+                    <td class="fix ex">signed_byte(1)</td>
                     <td>Function</td>
                   </tr>
                   <tr>
                     <td class="fix nt"></td>
                     <td class="fix op">|</td>
-                    <td class="fix ex">"\x02"</td>
+                    <td class="fix ex">signed_byte(2)</td>
                     <td >Binary (Old)</td>
                   </tr>
                   <tr>
                     <td class="fix nt"></td>
                     <td class="fix op">|</td>
-                    <td class="fix ex">"\x03"</td>
+                    <td class="fix ex">signed_byte(3)</td>
                     <td >UUID (Old)</td>
                   </tr>
                   <tr>
                     <td class="fix nt"></td>
                     <td class="fix op">|</td>
-                    <td class="fix ex">"\x04"</td>
+                    <td class="fix ex">signed_byte(4)</td>
                     <td>UUID</td>
                   </tr>
                   <tr>
                     <td class="fix nt"></td>
                     <td class="fix op">|</td>
-                    <td class="fix ex">"\x05"</td>
+                    <td class="fix ex">signed_byte(5)</td>
                     <td>MD5</td>
                   </tr>
                   <tr>
                     <td class="fix nt"></td>
                     <td class="fix op">|</td>
-                    <td class="fix ex">"\x06"</td>
+                    <td class="fix ex">signed_byte(6)</td>
                     <td><a href="https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/subtype6.rst">Encrypted BSON value</a></td>
                   </tr>
                   <tr>
                     <td class="fix nt"></td>
                     <td class="fix op">|</td>
-                    <td class="fix ex">"\x07"</td>
+                    <td class="fix ex">signed_byte(7)</td>
                     <td>Compressed BSON column</td>
                   </tr>
                   <tr>
                     <td class="fix nt"></td>
                     <td class="fix op">|</td>
-                    <td class="fix ex">"\x80"</td>
-                    <td >User defined</td>
+                    <td class="fix ex">signed_byte(128)</td>
```

Comment:
Do we need to do anything to represent that any subtype on the internal \[128, 255\] is reserved as user-defined? That's explained below, but seems to be documented improperly here. Not sure the best way to notate this, though.